tests/Api/Responses/Expected/admin/catalog_promotion/get_catalog_promotions_admin_response.json

```
@@ -6,7 +6,7 @@
         {
             "@id": "\/api\/v2\/admin\/catalog-promotions\/mugs_discount",
             "@type": "CatalogPromotion",
-            "id": @integer@,
+            "id": "@integer@",
```

Why did you change that? IMO here and in other responses there should be:
```suggestion
            "id": @integer@,
```